GenderGapsInCollegeDegrees.md

```diff
@@ -0,0 +1,160 @@
+
+## Clean up the chart
+1. Let's explore the gender gap in STEM fields, such as science, technology, engineering, and mathematics.
+2. First off, start with the historical percentage of Biology degrees for both genders.
+
+
+```python
+import pandas as pd
+import matplotlib.pyplot as plt
+#Read data
+women_degrees = pd.read_csv('percent-bachelors-degrees-women-usa.csv')
+#Visualize the gender gap of Biology degree
+fig, ax = plt.subplots()
+ax.plot(women_degrees['Year'], women_degrees['Biology'], c='red', label='Women')
+ax.plot(women_degrees['Year'], 100-women_degrees['Biology'], c='blue', label='Men')
+#Disable tick marks
+ax.tick_params(bottom="off", top="off", left="off", right="off")
+#Erase spines
+for key,spine in ax.spines.items():
+    spine.set_visible(False)
+#Display the chart
+ax.legend(loc='upper right')
+plt.show()
+```
+
+
+![png](output_1_0.png)
+
+
+## Compare across STEMs
+1. Let's compare gender gaps across 6 degrees.
```

Comment:
Linking this to the previous section a little more would be good.  ie "We've explored the gender gap in Biology degrees -- now let's look at some more subjects.  This will help us....".